### PR TITLE
Introduce customer portfolio with Cloud Trade as first customer

### PR DESCRIPTION
**Pull request by tomkerkhove**
Signed-off-by: Tom Kerkhove <kerkhove.tom@gmail.com>

### CODE REVIEW

**Comment by jeffhollan** [comment]
Word users is probably a bit more fitting instead of customers. Let me look around what other open projects refer to users as

**Comment by jeffhollan** [comment]
Yeah Prometheus says “some of our users include.”  Like that phrasing or something like it.

**Comment by tomkerkhove** [comment]
Sure, my pleasure!

File: content/_index/users/index.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
++++
+date = "2018-07-09"
+fragment = "portfolio"
+weight = 300
+background = "light"
+
+title = "Users"
+subtitle = "A variety of using Kubernetes Event-driven Autoscaling (KEDA) users are:"
```
Comment (zroubalik) [inline content/_index/users/index.md]: I'd rephrase the subtitle a little bit, "using", "users".

Comment (tomkerkhove) [reply content/_index/users/index.md]: Something like this then?

`A variety of users are autoscaling applications with Kubernetes Event-driven Autoscaling (KEDA):`

Comment (zroubalik) [reply content/_index/users/index.md]: Yeah, sounds good!
